.rubocop_todo.yml

```diff
@@ -191,6 +191,10 @@ RSpec/VerifiedDoubles:
     - 'spec/custom_facts/util/fact_spec.rb'
     - 'spec/custom_facts/util/resolution_spec.rb'
     - 'spec/facter/facts/aix/ssh_spec.rb'
+    - 'spec/facter/facts/freebsd/memory/swap/capacity_spec.rb'
+    - 'spec/facter/facts/freebsd/memory/swap/used_bytes_spec.rb'
+    - 'spec/facter/facts/freebsd/memory/system/capacity_spec.rb'
+    - 'spec/facter/facts/freebsd/memory/system/used_bytes_spec.rb'
```

Comment:
We should fix these offences. An example for `spec/facter/facts/freebsd/memory/swap/capacity_spec.rb`
```ruby
      expected_fact = instance_double(Facter::ResolvedFact, name: 'memory.swap.capacity', value: 1024)
```